packages/shared/src/components/auth/AuthOptions.tsx

```diff
@@ -258,6 +284,15 @@ function AuthOptions({
       }
 
       if (!e.data?.social_registration) {
+        const { data: boot } = bootResponse;
```

Comment:
Oh and here as well.

Reply:
Yep, we need these calls as our flow sometimes goes through registration flow even if it should be a login flow. However since it was identified to be a registered user, the end result is proving the same. So to catch all ends, we have to cover the callbacks.